.github/workflows/integration-tests-rootcheck-tier-0.yml

```diff
@@ -0,0 +1,86 @@
+name: Rootcheck integration tests tier 0
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch'
+        required: true
+        default: 'main'
+
+  pull_request:
+    paths:
```

Comment:
What about a trigger from the source code?

Reply:
I'm not sure, what you mean by this, but now that I moved the test to the *analysisd*, should I keep this here or we could use the analysisd to trigger this test also?